ic-os/components/conformance-tests/defs.bzl

```diff
@@ -21,7 +18,7 @@ def component_conformance_test(name, component_files, image):
 
     native.sh_test(
         name = name,
-        srcs = ["//ic-os/components/conformance:check_components.py"],
+        srcs = ["//ic-os/components/conformance-tests:check_file_references.py"],
```

Comment:
Could we use _ for consistency with most other package names? I believe we have both _ and - but _ is more prevalent.

Reply:
Sounds good! Renamed